content/blog/2022/12/2022-12-15-run-jenkins-agent-as-a-service.adoc

```diff
@@ -0,0 +1,362 @@
+----
+
+In my case, these packages were already installed and up to date.
+The next step is to add Docker’s official GPG key:
+
+[source,bash]
+----
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+----
+
+Then, we can set up the repo:
+
+[source,bash]
+----
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+----
+
+The last thing to do is to update the list of available packages, and then install the latest version of Docker:
+
+[source,bash]
+----
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin
+----
+
+If you’re - like me - running a recent version of Ubuntu or Debian, you won’t need to create the `docker` group, because it has been created with the installation of Docker.
+On the contrary, you can then issue a `sudo groupadd docker` command to create the `docker` group.
+
+Now, let’s add our current user to the `docker` group:
+
+[source,bash]
+----
+sudo usermod -aG docker $USER
+----
+
+And if you’re not using the default user, but `jenkins`, you can do the same:
+
+[source,bash]
+----
+sudo usermod -aG docker jenkins
+sudo usermod -aG sudo jenkins
+----
+
+Now log out, and log back in so that your group membership is updated.
+If you get any error, just reboot the machine, this sometimes happens.
+`¯\_(ツ)_/¯`
+
+Mandatory ``Hello World!'' Docker installation test:
+
+[source,bash]
+----
+docker run hello-world
+Unable to find image 'hello-world:latest' locally
+latest: Pulling from library/hello-world
+2db29710123e: Pull complete
+Digest: sha256:53f1bbee2f52c39e41682ee1d388285290c5c8a76cc92b42687eecf38e0af3f0
+Status: Downloaded newer image for hello-world:latest
+
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (amd64)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
+
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
+
+Share images, automate workflows, and more with a free Docker ID:
+ https://hub.docker.com/
+
+For more examples and ideas, visit:
+ https://docs.docker.com/get-started/
+----
+
+Nice!
+
+== Create a new node in Jenkins
+
+Quoting the official https://www.jenkins.io/doc/book/managing/nodes/[documentation],
+
+> Nodes are the ``machines'' on which build agents run.
```

Comment:
```suggestion
> Nodes are the "machines" on which build agents run.
```

Uses double quotes instead of a mix of backticks and simple quotes

Reply:
My eyes aren't anymore what they used to be. 🤣 
Thanks.